oracle/tests/functions/constructor.rs

```diff
@@ -0,0 +1,42 @@
+use crate::utils::{abi_calls::{constructor, owner}, test_helpers::setup, Identity};
+use fuels::{signers::Signer, tx::Address};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn construct() {
+        let user = setup().await;
+        assert!(false);
+        constructor(
+            &user.oracle,
+            Identity::Address(Address::from(user.wallet.address())),
+        )
+        .await;
+        let owner = owner(&user.oracle).await;
+        assert_eq!(
+            owner,
+            Option::Some(Identity::Address(Address::from(user.wallet.address())))
```

Comment:
Small nitpick but using a variable to hold the `Identity` would make this a lot more readable and can be used on line 13 as well. Same applies for other files.

Reply:
Whoops that file shouldn't even be included